### PR TITLE
Submit mismatches from Results page

### PR DESCRIPTION
**Pull request by guergana**
Bug: [T292054](https://phabricator.wikimedia.org/T292054)

### CODE REVIEW

**Review by Silvan-WMDE — COMMENTED** [review]
Great stuff - you made it work! \o/ :100:
I only have a few comments below, no blockers really.

File: tests/Vue/Pages/Results.spec.js [hunk 1]
```diff
@@ -147,4 +147,53 @@ describe('Results.vue', () => {
...
+                external_value: 'Another Value',
+                review_status: 'pending',
+                import_meta: {
+                    user: {
+                        username: 'some_user_name'
+                    },
+                    created_at: '2021-09-23'
+                },
+            }]
+        };
```
[the remark this answers is not in the document]
Comment (guergana) [reply tests/Vue/Pages/Results.spec.js:170]: oh, yes, you are right.

File: tests/Vue/Pages/Results.spec.js [hunk 2]
```diff
@@ -147,4 +147,53 @@ describe('Results.vue', () => {
...
+
+        const decisions = { [item_id]:{1:{id:1, item_id ,review_status: ReviewDecision.Wikidata}}};
+
+        const inertiaPut = jest.fn();
+
+        const wrapper = mount(Results, {
+            propsData: { results },
```
[the remark this answers is not in the document]
Comment (guergana) [reply tests/Vue/Pages/Results.spec.js:177]: yes!! thanks for pointing it out

File: tests/Vue/Pages/Results.spec.js [hunk 3]
```diff
@@ -147,4 +147,53 @@ describe('Results.vue', () => {
...
+        const decisionsBeforeDelete = decisions[item_id];
+        wrapper.vm.send( item_id );
+     
+        expect( inertiaPut ).toHaveBeenCalledWith( '/mismatch-review' , decisionsBeforeDelete );
+        
+        //the decisions object will be empty after sending the put request on one item
+        expect(wrapper.vm.decisions).toEqual({});
+
+    });
+
```
[the remark this answers is not in the document]
Comment (guergana) [reply tests/Vue/Pages/Results.spec.js:198]: I wouldn't think it is strictly necessary to had it but it doesn't hurt either :grin:

**Pull request merged**